src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/miner/IndustrialMiner.java

```diff
@@ -126,19 +126,50 @@ protected void registerDefaultFuelTypes() {
 
         Random random = ThreadLocalRandom.current();
 
+        if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_17)) {
+            // In 1.17, breaking metal ores should get raw metals. Also support deepslate ores.
+            switch (ore) {
+                case DEEPSLATE_COAL_ORE:
+                    return new ItemStack(Material.COAL);
```

Comment:
can't coal drop more than 1?

Reply:
This is the same as coal ore.

Reply:
@Mooy1 Only if you use fortune, coal ore always only dropped 1 coal item.

Minecraft wiki:
> "Coal ore mined using a pickaxe drops 1 unit of coal."